src/main/java/marquez/common/models/Urn.java

```diff
@@ -27,7 +27,8 @@
 @EqualsAndHashCode
 @ToString
 public abstract class Urn {
-  @Getter private final String value;
+
+  @Getter protected final String value;
```

Comment:
We'll need to be careful on merges. Looks like the scope of `value` has changed? Mind comparing you `Urn.java` with what's in master?

Reply:
This happened because we needed to use `value` in the subclass and modified it to be protected. Later, when I removed the methods that used it, I didn't adjust the scope back to being private.